src/Core/Managed/Shared/Extensibility/Implementation/BroadcastProcessor.cs

```diff
@@ -0,0 +1,148 @@
+﻿namespace Microsoft.ApplicationInsights.Shared.Extensibility.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
+
+    /// <summary>
+    /// An <see cref="ITelemetryProcessor"/> that asynchronously sends the data to multiple child telemetry processors.
+    /// </summary>
+    internal class BroadcastProcessor : ITelemetryProcessor, IDisposable
```

Comment:
My concern this brings complexity and make the behavior unclear. With this implementation you cannot implement the sink that will slow down `Track` so sending will keep up with the load if you need to implement back-pressure design for guarantee delivery instead of dropping items.

Code is good and can be used as a base class for event flow outputs.